### PR TITLE
XOR-430 [FEAT] JS documentation added for Batch endpoint for communicate(email and SMS ) and push notification endpoint also REST API documention added for Batch endpoint for communicate(email and SMS )

### PR DESCRIPTION
**Pull request by SMohol**
### JS API documentation
Documention added for JS API for batch email, SMS, and notification 

### REST API documentation
Documention added for REST API for batch email, SMS, and notification

### CODE REVIEW

**Review by yogeshmsharma-xoriant — DISMISSED** [review]
looks ok to me

File: docs/REST-API/fliplet-communicate.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+## Authentication
+
+Please head to the [how to authenticate](authenticate.md) page of the documentation to read more about how you can authorize your client to make API requests to Fliplet.
+
+---
+
+## App communicate
+
+When dealing with app communicate, there's a few things you should keep in mind:
+  to add
```
Comment (squallstar) [inline docs/REST-API/fliplet-communicate.md:16]: to add?

File: docs/REST-API/fliplet-communicate.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+      "attachments": [],
+      "images": []
+  }
+```
+
+---
+
+### Send batch emails
+
+Use our APIs to send batch of email to one or more recipients. Note that this feature is rate limited and improper use will result in your account being flagged for suspension.You need to pass emails with array of object as shown below.
```
Comment (squallstar) [inline docs/REST-API/fliplet-communicate.md:64]: Missing space in "suspension.You"

Comment (squallstar) [inline docs/REST-API/fliplet-communicate.md:64]: array of "objects"

File: docs/REST-API/fliplet-communicate.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+{
+  "provider": "twilio",
+  "data": {
+    "from": "+123456789",
+    "to": "+123456789",
+    "body": "Hey!"
+  },
+  "options": {
+    "twilio_sid": 'AC81caaa94b3b84bb7ba9c3cd96bcb152a', // Your Account SID from www.twilio.com/console
```
Comment (squallstar) [inline docs/REST-API/fliplet-communicate.md:141]: JSON does not support single quotes

File: docs/REST-API/fliplet-communicate.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+    "twilio_auth_token": 'AUTH_TOKEN'                 // Your Auth Token from www.twilio.com/console
+  }
+}
+```
+
+---
+
+### send batch SMS
+
+Use our APIs to send batch of SMS to one or more recipients. Note that this feature is rate limited and improper use will result in your account being flagged for suspension.You need to pass emails with array of object as shown below.
```
Comment (squallstar) [inline docs/REST-API/fliplet-communicate.md:151]: Missing space in "suspension.You"

File: docs/REST-API/fliplet-communicate.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+  "options": {
+    "twilio_sid": 'AC81caaa94b3b84bb7ba9c3cd96bcb152a', // Your Account SID from www.twilio.com/console
+    "twilio_auth_token": 'AUTH_TOKEN'                 // Your Auth Token from www.twilio.com/console
+  }
+}
+```
+
+---
+
+### send batch SMS
```
Comment (squallstar) [inline docs/REST-API/fliplet-communicate.md:149]: uppercase "S" in Send

File: docs/REST-API/fliplet-communicate.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+
+Use our APIs to send batch of SMS to one or more recipients. Note that this feature is rate limited and improper use will result in your account being flagged for suspension.You need to pass emails with array of object as shown below.
+
+#### ``POST;/v1/communicate/sms/batch`
```
Comment (squallstar) [inline docs/REST-API/fliplet-communicate.md:153]: You added two ` and it's breaking the formatting

File: docs/REST-API/fliplet-communicate.md [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+{
+"sms" : [{
+  "provider": "twilio",
+  "data": {
+    "from": "+123456789",
+    "to": "+123456789",
+    "body": "Hey!"
+  },
+  "options": {
+    "twilio_sid": 'AC81caaa94b3b84bb7ba9c3cd96bcb152a', // Your Account SID from www.twilio.com/console
```
Comment (squallstar) [inline docs/REST-API/fliplet-communicate.md:170]: Double quotes

File: docs/REST-API/fliplet-communicate.md [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+    "twilio_auth_token": 'AUTH_TOKEN'                  // Your Auth Token from www.twilio.com/console
+  }
+},
+{
```
Comment (squallstar) [inline docs/REST-API/fliplet-communicate.md:174]: Invalid formatting

**Review by squallstar — CHANGES_REQUESTED** [review]

File: docs/REST-API/fliplet-communicate.md [hunk 9]
```diff
@@ -0,0 +1,186 @@
...
+  to add
+
+
+## Endpoints
+
+### Send an email
+
+Use our APIs to send an email to one or more recipients. Note that this feature is rate limited and improper use will result in your account being flagged for suspension.
+
+#### `POST;/v1/communicate/email`
```
Comment (squallstar) [inline docs/REST-API/fliplet-communicate.md:25]: What's this semicolon for? Please check the formatting used in the other pages.

File: docs/REST-API/fliplet-communicate.md [hunk 10]
```diff
@@ -0,0 +1,186 @@
...
+  }
+```
+
+---
+
+### Send batch emails
+
+Use our APIs to send batch of email to one or more recipients. Note that this feature is rate limited and improper use will result in your account being flagged for suspension.You need to pass emails with array of object as shown below.
+
+#### `POST;/v1/communicate/email/batch`
```
Comment (squallstar) [inline docs/REST-API/fliplet-communicate.md:66]: What's this semicolon for? Please check the formatting used in the other pages.

File: docs/REST-API/fliplet-communicate.md [hunk 11]
```diff
@@ -0,0 +1,186 @@
...
+    ]
+}
+```
+
+---
+### Send an SMS
+
+Use our APIs to send an sms to one or more recipients. Note that this feature is rate limited and improper use will result in your account being flagged for suspension.
+
+#### `POST;/v1/communicate/sms`
```
Comment (squallstar) [inline docs/REST-API/fliplet-communicate.md:127]: What's this semicolon for? Please check the formatting used in the other pages.

File: docs/REST-API/fliplet-notifications.md [hunk 12]
```diff
@@ -147,6 +147,54 @@ You can also target many people at once using any [Sift.js](https://github.com/F
...
+    "message": "John posted new article."
+  },
+  "scope": { "Email": "nick@example.org" },
+  "pushNotification": {
+    "payload": {
+      "title": "Next article",
+      "body": "John has posted a next article on the news page. Go check it out!"
+    }
+  }
+}]
```
Comment (squallstar) [inline docs/REST-API/fliplet-notifications.md:194]: Invalid formatting

File: docs/REST-API/fliplet-notifications.md [hunk 13]
```diff
@@ -147,6 +147,54 @@ You can also target many people at once using any [Sift.js](https://github.com/F
...
+      "title": "New article",
+      "body": "John has posted a new article on the news page. Go check it out!"
+    }
+  }
+},{
+  "data": {
+    "title": "Next article",
+    "message": "John posted new article."
+  },
+  "scope": { "Email": "nick@example.org" },
```
Comment (squallstar) [inline docs/REST-API/fliplet-notifications.md:187]: You're demonstrating sending an array, so please use a different email? e.g. "jane@example.org"

**Review by squallstar — CHANGES_REQUESTED** [review]

**Review by squallstar — APPROVED** [review]

**Pull request merged**